wiki/News_styling_criteria/en.md

```diff
@@ -36,64 +32,93 @@ Consider well what you write within your article and whether it would interest t
+News articles that use images have their own folder within the [`wiki/shared/news/` directory](https://github.com/ppy/osu-wiki/tree/master/wiki/shared/news), with the same name as the news post file name. Example: `wiki/shared/news/2017-08-17-news-post-title/banner.jpg`.
 
-#### Formats, quality and file sizes
+### Banners
 
-Please refer to the [article styling criteria](/wiki/Article_styling_criteria/Formatting#formats-and-quality) wiki page.
+News posts must have an image after the preview paragraph for use as a **banner** (also called *cover*). The first image in an article will be used as its banner on the front page, news listing, and news article view.
+
+These banners appear in many different aspect ratios in different places of the website and on different devices, and should therefore be designed with any cropping due to this in mind.
+
+![News post banner aspect ratio reference](img/banner-sizes.jpg "News post banner aspect ratio reference. Note that mobile devices may fall anywhere within or even slightly outside the common ranges specified here.")
+
+Banners should have a minimum 1000x200 px base size. A version with each dimension doubled should be provided if the source image is large enough to support it (resulting in `banner.jpg` and `banner@2x.jpg`). Banner images appearing in multiple news articles should be put in the [`wiki/shared/news/banners/` directory](https://github.com/ppy/osu-wiki/tree/master/wiki/shared/news/banners).
 
 ### HTML and embedded content
 
-Limited HTML use for the purposes of embedding off-site content such as YouTube videos, Twitch VODs or other applets designed to showcase osu! or osu! related content is allowed.
+Limited use of HTML is allowed for the purposes of embedding off-site content such as YouTube videos, Twitch VODs, or other applets designed to showcase osu! or osu!-related content.
+
+The width of all embedded content frames must be set to 95% with the exception of tall aspect ratios. To be included in articles in general, embedded content must support display at full-width styling without breaking or looking awful.
 
-``<center>`` tag use (or any equivalent styling such as ``<p align='center'>``) is disallowed due to the news system styling automatically centring non-text elements.
+Video embed hosted on `assets.ppy.sh`:
 
-The width of all embedded content frames must be set to ``width: 95%``. To be included in articles in general, embedded content must support display at full-width styling without breaking or looking awful.
+```html
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/172/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+```
+
+Video embed hosted on YouTube:
+
+```html
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cXkiX7u4a9g" frameborder="0" allowfullscreen></iframe>
+</div>
```

Comment:
this `aspect-ratio` property is the most concise thing i could come up with to make these iframes behave. it's [fully supported on all major browsers since 2021](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#browser_compatibility) so all should be good

as mentioned in https://github.com/ppy/osu-wiki/pull/8994 i'll likely update previous news posts sometime soon